index.html

```diff
@@ -196,9 +286,30 @@ <h1>About
         <li class='menu'>Skills<span class='arrow rarrow'></span><span class='arrow darrow'>-</span></li>
       </ul>
 
+    </div>
+
+  </section>
+  <section class="contactform">
+    <div class="container"></div>
+    <div class="content-form">
+      <h3>Contact me</h3>
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that
+        needs coding. I’d love to help with it </p>
+      <form id="fs-frm" name="simple-contact-form" accept-charset="utf-8" action="https://formspree.io/f/xgerlrpz"
+        method="POST">
+        <fieldset id="fs-frm-inputs">
+          <input type="text" name="name" id="full-name" placeholder="Full name" maxlength="30" required>
+          <input type="email" name="_replyto" id="email-address" placeholder="Email address" required>
+          <textarea rows="5" name="message" id="message" placeholder="Enter text here" maxlength="500"
```

Comment:
- [ ] Kindly let's give to the `<textarea>` the same font-family as the one in the design, because for now it has the default font.